libraries/bot-builder/src/main/java/com/microsoft/bot/builder/skills/SkillHandlerImpl.java

```diff
@@ -68,13 +70,25 @@ public SkillHandlerImpl(
         this.getOAuthScope = withGetOAuthScope;
     }
 
+    /**
+     *
+     * @param claimsIdentity claimsIdentity for the bot, should have
+     *                       AudienceClaim, AppIdClaim and ServiceUrlClaim.
+     * @param conversationId conversationId.
+     * @param activity       Activity to send.
+     *
+     * @return Task for a resource response.
+     */
     public CompletableFuture<ResourceResponse> onSendToConversation(
         ClaimsIdentity claimsIdentity,
         String conversationId,
         Activity activity) {
         return processActivity(claimsIdentity, conversationId, null, activity);
     }
 
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
This class is not extending from another.. so the `inheritDoc` it's not okay. We should populate the commentaries.

Reply:
The "Impl" part really made me fall for it

Reply:
sometime later we should fix [this one](https://github.com/southworks/botbuilder-java/blob/internal/feature/southworks/cloudadapter/base/libraries/bot-builder/src/main/java/com/microsoft/bot/builder/skills/CloudSkillHandler.java#L140) too

Reply:
But that one is extending from the `CloudChannelServiceHandler`

Reply:
yes but the method is not there, or in the parent, or anywhere. Well, I didn't find it